test-harness/specs/parameters-ac7.oas2.txt

```diff
@@ -0,0 +1,29 @@
+====test====
+Request form-data body should be an array of strings (comma separated values)
```

Comment:
This test only proves there is not an error when `--data "arr=a,b,c"` but it doesn't prove there is an error when --data "arr=a/b/c" or something else. If we can try to show a positive and a negative it gives better confidence that the code is doing what it is meant to, otherwise it could just be a coincidence. "Not blowing up" != "works correctly".

Reply:
Good point, adding this now!